src/background-service.ts

```diff
@@ -25,7 +25,7 @@ import { getWelcomePageURL } from './extension/options-page/Welcome/getWelcomePa
 import { exclusiveTasks } from './extension/content-script/tasks'
 import { Flags } from './utils/flags'
 
-if (Flags.matrix_based_service_enabled) {
+if (process.env.NODE_ENV === 'development') {
```

Comment:
`Flags.matrix_based_service_enabled && process.env.NODE_ENV === 'development'` to use both flag and tree shaking

Reply:
write a NODE_ENV call detect scripts.
make CI failed?